src/Anexia.E5E/Functions/E5EEvent.cs

```diff
@@ -63,15 +64,43 @@ public record E5EEvent(E5ERequestDataType Type,
 	}
 
 	/// <summary>
-	///     Returns the value as byte enumerable.
+	///     Returns the bytes of the attached file.
 	/// </summary>
 	/// <exception cref="E5EInvalidConversionException">
-	///     Thrown if <see cref="Type" /> is not
-	///     <see cref="E5ERequestDataType.Binary" />.
+	///     Thrown if <see cref="Type" /> is not <see cref="E5ERequestDataType.Binary" />.
 	/// </exception>
-	public byte[]? AsBytes()
+	/// <exception cref="E5EMultipleFilesInFormDataException">Thrown if there are multiple files attached to this request.</exception>
+	public IEnumerable<byte> AsBytes()
+	{
+		E5EInvalidConversionException.ThrowIfNotMatch(E5ERequestDataType.Binary, Type);
+		if (Data.GetValueOrDefault().ValueKind == JsonValueKind.Array)
+			throw new E5EMultipleFilesInFormDataException();
+
+#if NET8_0_OR_GREATER
+		return As(E5ESerializationContext.Default.E5EFileData)?.GetBytes() ?? Enumerable.Empty<byte>();
+#else
+		return As<E5EFileData>()?.GetBytes() ?? Enumerable.Empty<byte>();
+#endif
+	}
+
+	/// <summary>
+	///     If this request is a multipart/form-data request, all files attached to this request are deserialized.
+	/// </summary>
+	/// <returns></returns>
+	/// <exception cref="NotImplementedException"></exception>
+	public IEnumerable<E5EFileData> AsFiles()
 	{
 		E5EInvalidConversionException.ThrowIfNotMatch(E5ERequestDataType.Binary, Type);
```

Comment:
Question: Is this the actual behavior of e5e? If I [understand the documentation correct](https://engine.anexia-it.com/docs/en/module/e5e/user-guide/input-and-output-formats), `multipart/form-data` would end up in a type `mixed`, but not in a `binary` type.

Reply:
For `multipart/form-data` requests, the event type is set to `"mixed"`. Each file object in the `data` list then has the `type` set to `"binary"`.